pkg/config/config.go

```diff
@@ -186,7 +186,7 @@ func (lci *LocalConfigInfo) SetConfiguration(parameter string, value interface{}
 		case "sourcelocation":
 			lci.componentSettings.SourceLocation = &strValue
 		case "ports":
-			arrValue := value.([]string)
+			arrValue := strings.Split(strValue, ",")
```

Comment:
wouldn't it be better to convert this huge switch case to separate set functions to avoid using `interface{}` like we have with get*?

Reply:
sure, we can have an issue which scopes out all the issues with `SetConfiguration`

Reply:
I think that the sooner we fix this properly the better

Reply:
a switch will always be there it will just be moved up in the set.go of the cli package

Reply:
lets record this in an issue